mama/c_cpp/src/testtools/capturereplay/c/capturereplayc.c

```diff
@@ -1,480 +1,632 @@
-static const char*              gFilename               = NULL;
-static const char*              gPubSource              = "WOMBAT";
-static const char**             gSymbolList             = NULL;
-static mamaTransport            gPubTransport           = NULL;
-static mamaDQPublisherManager   gDQPubManager           = NULL;
-static pubCache*                gSubscriptionList       = NULL;
-static int                      gNumSymbols             = 0;
-static MamaLogLevel             gSubscLogLevel          = MAMA_LOG_LEVEL_NORMAL;
-static mamaTimer                gPubTimer               = NULL;
-static double                   gTimeInterval           = 0.5;
-static int                      gRewind                 = 0;
-#define DELIM                   ':'
-
-static void parseCommandLine (int argc, const char** argv);
-
+static mamaBridge             gPubBridge              = NULL;
+static const char *           gPubMiddleware          = "wmw";
+static mamaQueue              gPubDefaultQueue        = NULL;
+static mamaSubscription       gDictionarySubscription = NULL;
+static mamaDictionary         gDictionary             = NULL;
+static const char *           gDictionaryFile         = NULL;
+static mamaMsg                gDictionaryMessage      = NULL;
+static mamaPublisher          gDictionaryPublisher    = NULL;
+static const char *           gPubTransportName       = "pub";
+static int                    gQuietness              = 0;
+static const char *           gFilename               = NULL;
+static const char *           gPubSource              = "WOMBAT";
+static const char **          gSymbolList             = NULL;
+static const char *           gSymbolListFile         = NULL;
+static mamaTransport          gPubTransport           = NULL;
+static mamaDQPublisherManager gDQPubManager           = NULL;
+static pubCache *             gSubscriptionList       = NULL;
+static int                    gNumSymbols             = 0;
+static MamaLogLevel           gSubscLogLevel          = MAMA_LOG_LEVEL_NORMAL;
+static mamaTimer              gPubTimer               = NULL;
+static double                 gTimeInterval           = 0.5;
+static int                    gRewind                 = 0;
+
+/* Set the header delimiter */
+#define DELIM ':'
+
+static void parseCommandLine (int argc, const char **argv);
+
 static void MAMACALLTYPE
-subscriptionHandlerOnCreateCb (mamaDQPublisherManager manager);
-
+subscriptionHandlerOnCreateCb (mamaDQPublisherManager manager);
 static void MAMACALLTYPE
-subscriptionHandlerOnNewRequestCb(mamaDQPublisherManager manager,
-                                  const char*            symbol,
-                                  short                  subType,
-                                  short                  msgType,
-                                  mamaMsg                msg);
+subscriptionHandlerOnNewRequestCb (mamaDQPublisherManager manager,
+                                   const char *           symbol,
```

Comment:
Space between type and asterisk is not our standard formatting - will need corrected as it's throughout the file.